scripts/MKW_SaveGhostToBothCSV.py

```diff
@@ -0,0 +1,24 @@
+from dolphin import gui
+import TTK_Lib
+
+"""
+MKW_SaveGhostToBothCSV
+
+This script takes the current ghost's inputs and write them to the ghost and player csvs
+"""
+
+def main() -> None:
+    gui.add_osd_message("Script started")
+    
+    # Convert internal RKG to input list
+    input_sequence = TTK_Lib.readFullDecodedRKGData(TTK_Lib.PlayerType.GHOST)
+    
+    if (input_sequence is None or len(input_sequence) == 0):
+        gui.add_osd_message("Not in race")
+        return
```

Comment:
For whatever reason, the scripting engine ignores this, so it isn't needed. We can use the other suggestion I made instead (applies to all relevant files).
```suggestion
```